proposals/wac-ucr/index.bs

```diff
@@ -42,5 +42,111 @@ This is where we explain the use case.
 Definitions {#definitions}
 =====================
 
-A <dfn>Definition</dfn> can be defined here and then referenced throughout
-the rest of the document.
+All definitions as stated below should be considered in the context of
+Web Access Control, whether explicitly stated or not.
+
+An <dfn>agent</dfn> is a distinct individual, group, organization, or piece
+of software with an [=identity=] that can be strongly authenticated.
+
+An <dfn>identity</dfn> is used to uniquely identify a given [=agent=],
+represented by a unique [[URI]]. Examples of compatible identity systems
+include [[WEBID]] and [[DID]].
+
+An <dfn>authenticated agent</dfn> is an [=agent=] that can prove control
+of a given [=identity=].
+
+An <dfn>application</dfn> is a piece of software that interfaces with a
+[=resource server=], which may operate autonomously as an
+[=authenticated agent=], or in piloted-fashion by another
+[=authenticated agent=].
+
+An <dfn>application identity</dfn> is used to uniquely identify a given
+[=application=], represented by a unique [[URI]].
+
+A <dfn>resource controller</dfn> is an [=agent=] with fully permissioned
+access and control over one or more [=resources=] residing on a
+[=resource server=] on the Web.
+
+A <dfn>resource server</dfn> is a networked system that provides an
+interface to make [=resources=] available to [=agents=] and [=applications=],
+with the ability to secure access to those [=resources=]
+through associated [=acl resources=].
+
+A <dfn>resource</dfn> is a set of data identified via a [[URI]].
```

Comment:
If not different than RFC 7231's "resource", refer to it instead of defining.

Reply:
Resolved in https://github.com/solid/authorization-panel/pull/83/commits/ccc047b647dc67c613248e9713c35bbbf20c826f.